web/src/engine/providers/BookmarkPlugin.ts

```diff
@@ -23,7 +25,7 @@ export class BookmarkPlugin extends MediaContainer<Bookmark> {
     }
 
     private Deserialize(serialized: BookmarkSerialized): Bookmark {
-        const parent = this.plugins.WebsitePlugins.find(plugin => plugin.Identifier === serialized.Media.ProviderID);
+        const parent = this.plugins.WebsitePlugins.find(plugin => plugin.Identifier === serialized.Media.ProviderID) ?? this._createInvalidParent('[DELETED] ' + serialized.Media.EntryID, serialized.Media.ProviderID);
```

Comment:
Name should probably also contain the ProviderID instead of the MangaID (which can be quite long and may not benefit the user)